src/components/plotly/pie/index.js

```diff
@@ -31,12 +41,16 @@ Pie.propTypes = {
   values: PropTypes.arrayOf(PropTypes.string).isRequired,
   donut: PropTypes.bool,
   showPercentage: PropTypes.bool,
+  showLabelName: PropTypes.bool,
+  extra: PropTypes.object,
```

Comment:
instead of exposing plotly internals like this, why not add more boolean props:

```
holeSize (default 0.4)
showValue (default false)
```

Reply:
yeah, I agree with Kyle!

Reply:
yea that works too